lib/jsdom/living/crypto/Crypto-impl.js

```diff
@@ -38,10 +37,30 @@ class CryptoImpl {
         "QuotaExceededError"
       ]);
     }
-
     nodeCrypto.randomFillSync(array);
     return array;
   }
 }
 
 exports.implementation = CryptoImpl;
+
+// See #3395. Subclasses of TypedArrays should properly work, but we can't rely
+// on instanceof because Uint8Array may be different across different windows -
+// which can happen in JSDOM when running { runScripts: "dangerously" }. As a
+// solution, we imitate the behavior of instanceof by walking the proottype
+// chain.
+function getChain(array) {
```

Comment:
It would be better for this function to have a clear name (I don't know what a "chain" is), like `getTypedArrayTypeName`, and for it to only return a string, not an array where its caller ignores the first value.